packages/html/ds/src/alert/alert-schema.ts

```diff
@@ -1,15 +1,18 @@
 import * as zod from 'zod';
+import { getEnumValues } from '../helpers';
 
-export enum AlertVariant {
-  INFO = 'info',
-  DANGER = 'danger',
-  SUCCESS = 'success',
-  WARNING = 'warning',
-}
+export const AlertVariant = {
+  INFO: 'info',
```

Comment:
I also noticed that sometimes we are using ALL_CAPS, and sometimes not.
Can we uniform this one too?